hooks/dot-bash.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+exit_code=0
+for file in "$@"; do
+	# TODO Confirm file has '.bash' extension
```

Comment:
can you simply use regex here?

Reply:
A quick googling confirms bash supports `if [[ $file == *.bash ]]`
I figured it would be easy, but was also wondering if we *want* the hook to confirm?  It might be better to leave the hook less opinionated and rely on the pre-commit filter:
* If we added support for different extensions in the future, the hook wouldn't need to be updated.
* If the pre-commit config was borked, having the check here might hide that fact

So my thoughts are to remove the TODO, but happy to turn it into an actual check if we think that's more valuable.